app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V1::CommentsController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @comments = Comment.where(post_id: params[:post_id]).order('created_at')
+    render json: { success: true, data: { comments: @comments } }
+  end
```

Comment:
- [ ] Your comments index api endpoint returns a 500 internal server error. Kindly fix this error, please.

Reply:
Hi @Kamba56 , Can you specifically show me the error you are getting? Because when I try to test it works fine. It'll be great if you explain me about the comments index API endpoint error. 
![Screenshot (66)](https://github.com/Rhaegar121/Blog-app/assets/111948881/33824144-b049-44eb-8db2-9130b52ce848)

Reply:
Hello @Rhaegar121, kindly submit for re-review. I don't know why I was getting the 500 error. If it works, then submit it for re-review.